src/test/e2e-emulated/map.spec.ts

```diff
@@ -1307,4 +1307,6345 @@ describe("map", () => {
+                    valueSmallAbs,
+                );
+                expect(await contract.getAddrMap7_6Value(addr)).toBe(
+                    valueSmallAbs,
+                );
+
+                // Sizes
+                expect((await contract.getIntMap1()).size).toBe(1);
+                expect((await contract.getIntMap2()).size).toBe(1);
+                expect((await contract.getIntMap3()).size).toBe(1);
+                expect((await contract.getIntMap4()).size).toBe(1);
+                expect((await contract.getIntMap5()).size).toBe(1);
+                expect((await contract.getIntMap6_1()).size).toBe(1);
+                expect((await contract.getIntMap6_2()).size).toBe(1);
+                expect((await contract.getIntMap6_3()).size).toBe(1);
+                expect((await contract.getIntMap6_4()).size).toBe(1);
+                expect((await contract.getIntMap6_5()).size).toBe(1);
+                expect((await contract.getIntMap6_6()).size).toBe(1);
+                expect((await contract.getIntMap6_7()).size).toBe(1);
+                expect((await contract.getIntMap7_1()).size).toBe(1);
+                expect((await contract.getIntMap7_2()).size).toBe(1);
+                expect((await contract.getIntMap7_3()).size).toBe(1);
+                expect((await contract.getIntMap7_4()).size).toBe(1);
+                expect((await contract.getIntMap7_5()).size).toBe(1);
+                expect((await contract.getIntMap7_6()).size).toBe(1);
+                expect((await contract.getIntMap8_1()).size).toBe(1);
+                expect((await contract.getIntMap8_2()).size).toBe(1);
+                expect((await contract.getIntMap8_3()).size).toBe(1);
+                expect((await contract.getIntMap8_4()).size).toBe(1);
+                expect((await contract.getIntMap8_5()).size).toBe(1);
+                expect((await contract.getIntMap8_6()).size).toBe(1);
+                expect((await contract.getIntMap8_7()).size).toBe(1);
+                expect((await contract.getIntMap9_1()).size).toBe(1);
+                expect((await contract.getIntMap9_2()).size).toBe(1);
+                expect((await contract.getIntMap9_3()).size).toBe(1);
+                expect((await contract.getIntMap9_4()).size).toBe(1);
+                expect((await contract.getIntMap9_5()).size).toBe(1);
+                expect((await contract.getIntMap9_6()).size).toBe(1);
+                expect((await contract.getIntMap10_1()).size).toBe(1);
+                expect((await contract.getIntMap10_2()).size).toBe(1);
+                expect((await contract.getIntMap10_3()).size).toBe(1);
+                expect((await contract.getIntMap10_4()).size).toBe(1);
+                expect((await contract.getAddrMap1()).size).toBe(1);
+                expect((await contract.getAddrMap2()).size).toBe(1);
+                expect((await contract.getAddrMap3()).size).toBe(1);
+                expect((await contract.getAddrMap4()).size).toBe(1);
```

Comment:
Is there a reason for the missing getAddrMap 5, etc.? This happens in several places below as well.

Reply:
just missed these randomly, will make sure to not miss anything in the refactor (it'll be much easier due to less repetitive code and better structure)